
---
int64 count
float64[] x
float64[] y
float64[] z
int32[] height
int32[] width 
